apps/studio/src/types/schema.ts

```diff
@@ -4,14 +4,24 @@ import {
   type UISchemaElement,
   type VerticalLayout,
 } from '@jsonforms/core'
+import { type IsomerComplexComponentProps } from '@opengovsg/isomer-components'
 
-export type IsomerJsonSchema = JsonSchema & {
+export type IsomerExtendedJsonSchema = JsonSchema & {
   groups?: Array<{
     label: string
     fields: string[]
   }>
 }
 
+export type IsomerJsonSchema = IsomerExtendedJsonSchema & {
+  components: {
+    complex: Record<
```

Comment:
how about native? Why do we only define selectively?

Reply:
Added native in f0f0d5e486b4f7899c3e54e2d3b9753bbee4a956. Opted not to define more than what we really need because that would start to defeat the purpose of having the schema outside of studio.

Eventually we will solve this with TypeBox when we are able to export the same types and schema from the components package.